Servers/Administration/OJS.Servers.Administration/Controllers/ContestsController.cs

```diff
@@ -200,7 +210,13 @@ await this.categoriesValidationHelper
                 newContest.Duration = null;
             }
 
-            newContest.IpsInContests.Clear();
+            var ipsInContests = this.GetContestIps(existingContest.Id);
+
+            if (ipsInContests.Count() != 0)
```

Comment:
You can also use .Any() for more simplicity.

Reply:
It's fixed